src/main/java/com/esri/arcgisruntime/toolkit/skins/FeatureTemplatePickerTilePaneSkin.java

```diff
@@ -289,6 +304,12 @@ private ToggleButton createToggleButtonForFeatureTemplateItem(FeatureTemplateIte
     toggleButton.setToggleGroup(toggleGroup);
     // bind the preferred width of the button based on the symbol size to limit overflowing template names
     toggleButton.prefWidthProperty().bind(getSkinnable().symbolSizeProperty().multiply(2));
+    // disable the toggle button if its feature layer is not editable
+    toggleButton.disableProperty().bind(Bindings.createBooleanBinding(() ->
+        getSkinnable().getDisableTemplatesFromUneditableFeatureLayers() &&
+            !featureTemplateItem.getFeatureLayer().getFeatureTable().isEditable(),
```

Comment:
I think maybe it should use `canAdd()` for this but you would probably need to rename the property too.

Reply:
@colinanderson Done. Let me know if you have a suggestion for a better property name.

Reply:
I think the name should reflect that is `canAdd` so something like `disableTemplatesWhenCanAddNotSupported` or something. I think it will be pretty difficult to come up with a short name for this.

Reply:
Maybe `disableWhenCannotAddProperty' is enough and the doc would says something along the line of "Disables templates when the feature service does not support adding new features".